gulpfile.js

```diff
@@ -185,6 +185,19 @@ function createWebpackConfig(defines, output) {
     !bundleDefines.TESTING;
   var skipBabel = bundleDefines.SKIP_BABEL;
 
+  // `core-js` (see https://github.com/zloirock/core-js/issues/514),
+  // `web-streams-polyfill` (already using a transpiled file), and
+  // `src/core/{glyphlist,unicode}.js` (Babel is too slow for those when
+  // source-maps are enabled) should be excluded from processing.
+  const babelExcludes = [
+    "node_modules[\\\\\\/]core-js",
```

Comment:
I'm going to assume that, like the second commit, the number of backslashes here is correct, although it looks a bit much :-)

Reply:
For `gulp mozcentral` the resulting RegExp looks as follows, which is consistent with the old format:
```
/(node_modules[\\\/]core-js|node_modules[\\\/]web-streams-polyfill)/
```
For `gulp generic` the resulting RegExp looks as follows, which is also consistent with the old format:
```
/(node_modules[\\\/]core-js|node_modules[\\\/]web-streams-polyfill|src[\\\/]core[\\\/](glyphlist|unicode))/
```

Since we now build the regular expressing using "normal" strings, we need to do more escaping to not lose part of it.